meson.build

```diff
@@ -23,9 +23,9 @@ mandir     = join_paths(prefixdir, get_option('mandir'))
 bindir     = join_paths(prefixdir, get_option('bindir'))
 sysconfdir = join_paths(prefixdir, get_option('sysconfdir'))
 
-udevrulesdir   = join_paths(prefixdir, get_option('udevrulesdir'))
-dracutrulesdir = join_paths(prefixdir, get_option('dracutrulesdir'))
-systemddir     = join_paths(prefixdir, get_option('systemddir'))
+udevrulesdir   = join_paths(sysconfdir, get_option('udevrulesdir'))
+dracutrulesdir = join_paths(libdir, get_option('dracutrulesdir'))
+systemddir     = join_paths(libdir, get_option('systemddir'))
```

Comment:
Hmm, let me test this. It took a while to figure out this part. But I don't have really a clue when it comes to this :)

Reply:
I think you should not change this part here. For example, when I set the systemd unit directory via
  ` -Dsystemddir=%{_unitdir} `
then it will be prefixed with `sysconfdir`. Though in this example `%{_unitdir}` is an absolute path. This is way the default value is `etc/udev/rules.d` and not `udev/rules.d`. This gives the packager the freedom to define the complete path.

Reply:
https://build.opensuse.org/package/live_build_log/openSUSE:Factory/nvme-cli/standard/x86_64

```
[   50s]   User defined options
[   50s]     auto_features : enabled
[   50s]     bindir        : /usr/bin
[   50s]     buildtype     : plain
[   50s]     datadir       : /usr/share
[   50s]     includedir    : /usr/include
[   50s]     infodir       : /usr/share/info
[   50s]     libdir        : /usr/lib64
[   50s]     libexecdir    : /usr/libexec
[   50s]     localedir     : /usr/share/locale
[   50s]     localstatedir : /var
[   50s]     mandir        : /usr/share/man
[   50s]     prefix        : /usr
[   50s]     sbindir       : /usr/sbin
[   50s]     sharedstatedir: /var/lib
[   50s]     sysconfdir    : /etc
[   50s]     wrap_mode     : nodownload
[   50s]     docs          : man
[   50s]     dracutrulesdir: /etc/dracut/dracut.conf.d
[   50s]     systemddir    : /usr/lib/systemd/system
[   50s]     udevrulesdir  : /usr/lib/udev/rules.d
```

Reply:
As you can see not even `systemddir` is under lib64 for openSUSE. You can't assume where those files need to be installed. This is distro specific stuff.

Reply:
If I am not completely misstaken `%{_udevrulesdir}`, `%{_udevrulesdir}` and `%{_unitdir}` should be defined for Fedora/RHEL systems as well. So you should be able to do this here (I do this for openSUSE):
```
%meson \
    -Dudevrulesdir=%{_udevrulesdir} \
    -Ddracutrulesdir=%{_sysconfdir}/dracut/dracut.conf.d \
    -Dsystemddir=%{_unitdir} \
    -Ddocs=man
```

Reply:
At the first, I was doing the same thing you gave me the option.
But I was thinking that just remove options since there is predefined values from meson.
But thought It might have reason to be added on option, so I modified like that.

Anyway, Thanks for the checking openSUSE
I will push modified version after get back to the office on next Thur.